From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Tue, 2 Mar 2021 18:58:04 +0100
Subject: [PATCH] mm: slub: Don't enable partial CPU caches on PREEMPT_RT by
 default

SLUB's partial CPU caches lead to higher latencies in a hackbench
benchmark.

Don't enable partial CPU caches by default on PREEMPT_RT.

Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
---
 init/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/init/Kconfig
+++ b/init/Kconfig
@@ -1983,7 +1983,7 @@ config SHUFFLE_PAGE_ALLOCATOR
 	  Say Y if unsure.
 
 config SLUB_CPU_PARTIAL
-	default y
+	default y if !PREEMPT_RT
 	depends on SLUB && SMP
 	bool "SLUB per cpu partial cache"
 	help
